### PR TITLE
Very basic file export

### PR DESCRIPTION
**Pull request by rjgoodin**
Tracks # infected cells in each leaf over time, and exports to a .csv file.

Really basic implementation so far, just wanted to share this code with Joyce.

Working in a side file for now -- leaf_network_robert_mod -- not the real/full model.

### CODE REVIEW

File: models/viralspread/netlogo/LeafNetwork_Robert_mod.nlogo [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+  total-leaf1
+  total-leaf2
+  total-leaf3
+  total-leaf4
+  infected-leaf1
+  infected-leaf2
+  infected-leaf3
+  infected-leaf4
+]
+
```
Comment (jrhawley) [inline models/viralspread/netlogo/LeafNetwork_Robert_mod.nlogo:12]: can we make these arrays or something? or does netlogo not do arrays?

Comment (rjgoodin) [inline models/viralspread/netlogo/LeafNetwork_Robert_mod.nlogo:12]: NetLogo can do arrays, yeah, though the primary data structure is lists. 
Arrays may be better when we have an aribitrary number leaves. But for now, just having a few global variables, updating their values and printing to file does what we want.

**Comment by mattsmart** [comment]
- "Working in a side file for now -- leaf_network_robert_mod -- not the real/full model."

why not add to main file?

if this is just for sharing code, should we merge it?

**Comment by jrhawley** [comment]
K, seems like it's more for sharing. Going to merge

**Pull request merged**